problem_builder/tests/integration/xml_templates/slider_step.xml

```diff
@@ -0,0 +1,16 @@
+<step-builder display_name="Step Builder" max_attempts="90">
```

Comment:
@bradenmacdonald Nit: No need to set `max_attempts` here, I think (tests don't use it).